vs-code/vscode.md

```diff
@@ -48,6 +49,28 @@ Most software is written using text editors. Visual Studio Code (VS Code) is a v
 
 [Slides: Local Development with VS Code](https://docs.google.com/presentation/d/1QAMxrS1ZOvtSn7MuYxTmeanrMQi_O8Z5UVq-4LzLqso/edit?usp=sharing)
 
+### VSCode Tips
+
+- **Auto Indent** - VS Code package to format your JavaScript / TypeScript / CSS using [Prettier](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode).
+
+- **Indent multiple lines** - Just highlight the lines you want, then press tab to indent the entire code block, or **shift+tab** to un-indent the entire code block.
+
+- **Code Folding** - Folds the innermost uncollapsed region at the cursor.<br>
+                     **"Ctrl + Shift + ["** on Windows and Linux.<br>
+                     **"⌥ + ⌘ + ["** on macOS.
+                     
+- **Autosave** - The easiest way to turn on **Auto Save** is with the **File > Auto Save** toggle that turns on and off save after a                      delay.                    
```

Comment:
This sentence seems incomplete, is there a missing conjunction?

Reply:
Done.